infrastructure/cli/initializer.go

```diff
@@ -46,13 +47,15 @@ func NewInitializer(errorReporter error_reporting.ErrorReporter, installer insta
 }
 
 func (i *Initializer) Init() error {
+	const errorMessage = "could not download CLI" // Returning an error notifies that the scan cannot continue
```

Comment:
"Could not initialize Snyk Language Server dependencies" may be more appropriate here, as we don't try to download when managed binaries are disabled.